cmat/output_generation/evaluation/ols_utils.py

```diff
@@ -33,7 +35,11 @@ def fetch_eval_data(*, db_iden=None, uri=None, include_neighbors=False):
     children = {}
 
     url = build_ols_query(ontology_uri)
-    json_response = json_request(url)
+    try:
+        json_response = json_request(url)
+    except RequestException:
+        logger.warning(f'OLS4 error for {url}, trying OLS3...')
```

Comment:
FYI @tcezard 3 URIs fell through to OLS3 on the full run, so I manually searched for the terms in both web interfaces to compare the results (especially presence in EFO and xrefs):
* `http://purl.obolibrary.org/obo/HP_0002180` - [OLS3](https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FHP_0002180) vs. [OLS4](https://www.ebi.ac.uk/ols4/ontologies/efo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FHP_0002180)
* `http://purl.obolibrary.org/obo/MONDO_0005015` - [OLS3](https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0005015) vs. [OLS4](https://www.ebi.ac.uk/ols4/ontologies/mondo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FMONDO_0005015)
* `http://purl.obolibrary.org/obo/HP_0032316` - [OLS3](https://www.ebi.ac.uk/ols/ontologies/hp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FHP_0032316) vs. [OLS4](https://www.ebi.ac.uk/ols4/ontologies/hp/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FHP_0032316)

I think it looks fine.

Reply:
I reported the first one, will add the other two to the issue in case they have the same root cause.